src/Purebred.hs

```diff
@@ -145,7 +147,11 @@ appconfig =
 optParser :: ParserInfo AppConfig
 optParser = info
   (appconfig <**> helper)
-  (fullDesc <> progDesc "purebred" <> header "a search based, terminal mail user agent")
+  (fullDesc
+   <> progDesc "purebred"
+   <> header ("a search based, terminal mail user agent - " <> versionString))
+  where
+    versionString = showVersion version
```

Comment:
We might as well add a `--version` option while we're at it?

Reply:
Fair enough... will update